src/ServiceFabric.Mocks/MockActorProxyFactory.cs

```diff
@@ -7,13 +7,29 @@
 
 namespace ServiceFabric.Mocks
 {
+    using Microsoft.ServiceFabric.Actors.Runtime;
     using System.Collections.Generic;
-
+          
+    public delegate void MisingActorEventHandler(object sender, MisingActorEventArgs args);
     /// <summary>
     /// Specifies the interface for the factory that creates proxies for remote communication to the specified Actor.
     /// </summary>
     public class MockActorProxyFactory : MockServiceProxyFactory, IActorProxyFactory
-    {
+    {       
+
+        public event MisingActorEventHandler MisingActor;
+
+        protected virtual void OnMisingActor(object sender, ActorId id)
+        {
+            var args = new MisingActorEventArgs()
+            {
+                Id = id
+            };
+
+            MisingActor?.Invoke(sender, args);
+        }
```

Comment:
Following earlier suggestion, you'd register the Actor here, if it was set by the event handler